app/domain/entity/service.js

```diff
@@ -30,6 +30,30 @@ function EntityService($rootScope, $http, $q) {
         });
     };
 
+
+    service.getByUuid = function(urlBase, responseClass, uuid) {
+        var url = $rootScope.restApiUrl + urlBase + '/' + uuid;
+
+        var responsePromise = $http.get(url, service.getHeaders());
+
+
+        return $q(function(resolve, reject) {
+            responsePromise.then(
+                function(response) {
+                    console.log("GOOD");
+                    var entities = response.data;
+                    resolve(entities);
+                },
+                function(reject){
+                    console.log(reject);
```

Comment:
I think this be function(response) { reject(response) } so we can pass it back to the controller and display it more nicely.